src/main/java/edu/kis/powp/jobs2d/visitor/MirrorFigureCommandVisitor.java

```diff
@@ -0,0 +1,21 @@
+package edu.kis.powp.jobs2d.visitor;
+
+import edu.kis.powp.jobs2d.command.*;
+
+public class MirrorFigureCommandVisitor implements TransformCommandVisitor{
+
+    private final boolean mirrorHorizontal;
+    private final boolean mirrorVertical;
+
+    public MirrorFigureCommandVisitor(boolean mirrorHorizontal, boolean mirrorVertical){
+        this.mirrorHorizontal = mirrorHorizontal;
+        this.mirrorVertical = mirrorVertical;
+    }
+
+    @Override
+    public void visitPositionCommand(PositionCommand command){
```

Comment:
Nice try. I appreciate the solution, but I am reluctant to perform merging. Assuming mutability to all commands is not acceptable. I argue that mutability makes the code hard to understand and errorprone. 
Please make another attempt without setters.